cmd/main.go

```diff
@@ -46,19 +50,24 @@ func init() {
 	//+kubebuilder:scaffold:scheme
 }
 
+const defaultExpirationTimeInHours = 7 * 12
+
 func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var probeAddr string
 	var gardenerKubeconfigPath string
+	var gardenerProjectName string
+	var expirationInHours int64
 
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&gardenerKubeconfigPath, "gardener-kubeconfig-path", "/gardener/kubeconfig/kubeconfig", "Kubeconfig file for Gardener cluster")
-
+	flag.StringVar(&gardenerProjectName, "gardener-project-name", "gardener-project", "Name of the Gardener project")
+	flag.Int64Var(&expirationInHours, "expiration-in-hours", defaultExpirationTimeInHours, "Dynamic kubeconfig expiration time in seconds")
```

Comment:
variable says about expiration in hours while the message says about seconds. Is it intended?

Reply:
Removed time units from the flag name. I used duration instead as it contains unit.